<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Inscriptionig</class>
 <widget class="QWidget" name="Inscriptionig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Incriptionig</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="1">
    <widget class="QLabel" name="NumeroLabel">
     <property name="text">
      <string>Numero</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0">
    <widget class="QLabel" name="RueLabel">
     <property name="text">
      <string>Rue</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="EmailLabel">
     <property name="text">
      <string>Adresse mail</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="Telephone"/>
   </item>
   <item row="12" column="1">
    <widget class="QLineEdit" name="Numero"/>
   </item>
   <item row="8" column="1">
    <widget class="QLineEdit" name="Email"/>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="PhoneLable">
     <property name="text">
      <string>Numéro de Téléphone</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0">
    <widget class="QLineEdit" name="Rue"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="VilleLabel">
     <property name="text">
      <string>Ville</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLineEdit" name="Ville"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="PrenomLabel">
     <property name="text">
      <string>Prénom</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="Nom"/>
   </item>
   <item row="2" column="0" alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="title">
     <property name="text">
      <string>GoodBasket</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLineEdit" name="Postal"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="PostaleLabel">
     <property name="text">
      <string>Code Postale</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="PassLabel">
     <property name="text">
      <string>Mot de Passe</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLineEdit" name="Pass"/>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="Prenom"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="NomLabel">
     <property name="text">
      <string>Nom </string>
     </property>
    </widget>
   </item>
   <item row="15" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
    <widget class="QPushButton" name="Inscrire">
     <property name="text">
      <string>Inscrire</string>
     </property>
    </widget>
   </item>
   <item row="15" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
    <widget class="QPushButton" name="Login">
     <property name="text">
      <string>Connexion</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
